.github/workflows/post-merge-matrix-deploy.yml

```diff
@@ -120,7 +125,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target: [ ADDRESS_BUILD, FRAUD_BUILD, KBV_BUILD, CIC_BUILD, DL_BUILD ]
+        target: [ ADDRESS_BUILD, FRAUD_BUILD, KBV_BUILD, CIC_BUILD, DL_BUILD, PASSPORT_BUILD ]
```

Comment:
Could we name this PASSPORTA_BUILD just so we know its different from legacy and so that it aligns with the current stack names

Reply:
Changed